### PR TITLE
Migrate critical constants and acentric factors batch 5

### PR DESCRIPTION
**Pull request by saikat-samanta**
TABLE 2-106

### CODE REVIEW

File: src/data/critical-constants-acentric-factor.ts [hunk 1]
```diff
@@ -1263,4 +1263,1300 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    criticalPressure: 4.25,
+    criticalVolume: 0.27,
+    criticalCompressibilityFactor: 0.258,
+    acentricFactor: 0.342296,
+  },
+
+  MethylAmine: {
+    name: 'Methyl amine',
+    formula: 'CH5N',
+    CAS: ' 74-89-5',
```
Comment (maneetgoyal) [inline src/data/critical-constants-acentric-factor.ts:2386]:
```suggestion
    CAS: '74-89-5',
```

**Review by maneetgoyal — APPROVED** [review]
Minor mistake

**Review by qur786 — APPROVED** [review]
All good

File: src/data/critical-constants-acentric-factor.ts (hunk 1 above)
Comment (saikat-samanta) [reply src/data/critical-constants-acentric-factor.ts]: done 👍

**Pull request merged**